internal/gatewayapi/runner/runner.go

```diff
@@ -231,6 +233,21 @@ func (r *Runner) subscribeAndTranslate(ctx context.Context) {
 	r.Logger.Info("shutting down")
 }
 
+func isEndpointRoutingDisabled(resources *gatewayapi.Resources) bool {
```

Comment:
in `internal/gateway/runner` we set `endpointRoutingDisabled: false` by default but in `/internal/cmd/egtl` we want to enable it by default to simplify the CLI operation